resources/assets/coffee/osu_common.coffee

```diff
@@ -59,6 +59,8 @@ along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
   parseJson: (id) ->
     JSON.parse document.getElementById(id).text
 
+  isInputElement: (el) ->
+    el.tagName == 'INPUT' or el.tagName == 'SELECT' or el.tagName == 'TEXTAREA' or el.isContentEditable;
```

Comment:
`el.tagName in ['INPUT', 'SELECT', 'TEXTAREA']` is also a nicer way to write it.
(The compiler unrolls it if the lhs is an array literal (and splatless))